DependencyInjection/Configuration.php

```diff
@@ -208,4 +210,56 @@ private function addVisitorsSection(NodeBuilder $builder)
             ->end()
         ;
     }
+
+    private function addContextSection(NodeBuilder $builder)
+    {
+        $builder
+            ->arrayNode('context')
```

Comment:
I supposed to use the same context for both serialization and deserialization purposes. But the idea with separate contexts is, of course, more flexible. So yes, I think I'll split them.